src/containers/Tasks/TaskBuilder/TaskBuilder.js

```diff
@@ -0,0 +1,125 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import * as actionTypes from '../../../store/actions/index';
+// import axios from '../../../axios-tasks';
+import { Redirect } from 'react-router-dom';
+
+import Aux from '../../../hoc/aux';
+
+import '../../../assets/semantic/semantic.min.css';
+import { Form, Input, Button } from 'semantic-ui-react';
+
+import Response from '../../../components/Response/Response';
+
+
+
+class TaskBuilder extends Component {
+    state = {
+        task: {
+            title: '',
+            description: ''
+        },
+        taskStatus: ''
+    };
+    componentDidMount() {
+        // console.log(this.props);
+    }
+
+    createTaskHandler = () => {
+        const task = {
+            ...this.state.task
+        }
+        this.props.onAddTasks(task, this.props.token);
+        const updatedState = {
+            ...this.state,
+            task: {
+                ...this.state.task,
+                title: '',
+                description: '',
+            },
+            taskStatus: 'success'
+        };
+        // console.log(updatedState);
+        this.setState({ updatedState });
+        // console.log(this.state);
+    };
+
+    titleChangeHandler = (event) => {
+        const updatedTask = {
+            ...this.state.task,
+            title: event.target.value
+        };
+        this.setState({ task: updatedTask });
+    };
+
+    descriptionChangeHandler = (event) => {
+        const updatedDescription = {
+            ...this.state.task,
```

Comment:
Optimise changeHandler
instead of different method try to do in a single method dynamically